grafana/server.sls

```diff
@@ -14,6 +14,16 @@ grafana_packages:
   - require:
     - pkg: grafana_packages
 
+/etc/grafana/ldap.toml:
```

Comment:
In fact ldap.toml is always installed because it is part of the package. So when you install the package you have this file. And I just generate the same file and avoid to add an if. But yes I can add an if...